3723/1/spec/page_printer_spec.rb

```diff
@@ -0,0 +1,16 @@
+require './page_printer'
+require './sorted_pages'
+
+RSpec.describe PagePrinter do
+  subject { PagePrinter.new(row) }
+  let(:row) { [%w[faraday 1], %w[rspec 1], %w[faraday 3], %w[rails 5], %w[faraday 7]] }
+
+  describe '#call' do
+    it 'can sort by number from top' do
```

Comment:
The spec expects count to be changed. If it sorts, you should define sorted rows manually and expect `subject.call` to be equal to those sorted rows